extensions/3DTILES_implicit_tiling/0.0.0/README.md

```diff
@@ -1,37 +1,849 @@
+          -0.0005682966577418737,
+          0.8987233516605286,
+          0.00011646582098558159,
+          0.8990603398325034,
+          0,
+          241.6
+        ]
+      },
+      "refine": "ADD",
+      "subdivisionScheme": "QUADTREE"
+    }
+  }
+}
+```
+## Tile Coordinates
+
+**Tile coordinates** are a tuple of integers that uniquely identify a tile. Tile coordinates are either `(level, x, y)` for quadtrees or `(level, x, y, z)` for octrees. All tile coordinates are 0-indexed.
+
+`level` is 0 for the root tile. The root tile's children are at level 1, and so on.
+
+`x`, `y`, and `z` coordinates are measured in tiles from a side of the root bounding volume at the current level along one of the coordinate axes. The type of bounding volume (`box` or `region`) determines the direction of increasing tile coordinates.
+
+For `box`, the tile coordinates are listed along the same direction as the Cartesian coordinate system of the tileset. That is:
+
+| Coordinate | Positive Direction |
+| --- | --- |
+| `x` | Along the `+x` axis of the bounding box |
+| `y` | Along the `+y` axis of the bounding box |
+| `y` | Along the `+z` axis of the bounding box |
+
+![Box coordinates](figures/box-coordinates.jpg)
+
+For `region` bounding volumes, the coordinates are interpreted in Cartographic space, as desribed in [the 1.0 specification](https://github.com/CesiumGS/3d-tiles/tree/master/specification#region). That is:
+
+| Coordinate | Positive Direction |
+|---|---|
+| `x` | From west to east (increasing longitude) |
+| `y` | From south to north (increasing latitude) |
+| `z` | From bottom to top (increasing height) |
+
+![Region Coordinates](figures/region-coordinates.jpg)
+
+## Template URIs
+
+A **Template URI** is a URI pattern used to refer to tiles by their tile coordinates.
+
+Template URIs are configured in the tileset.json. They may be any URI pattern, but must include the variables `${level}`, `${x}`, `${y}`. Template URIs for octrees must also include `${z}`. When referring to a specific tile, the tile's coordinates are substituted in for these variables.
+
+Here are some examples of template URIs and files that they match:
+
+```
+== Quadtree Example ==
+Pattern: "content/${level}/${x}/${y}/tile.b3dm"
+Valid filenames: 
+- content/0/0/0/tile.b3dm
+- content/1/1/0/tile.b3dm
+- content/3/2/2/tile.b3dm
+
+== Octree Example ==
+Pattern: "content/${level}/${z}/${y}/${x}.pnts"
+Valid filenames:
+- content/0/0/0/0.pnts
+- content/1/1/1/1.pnts
+- content/3/2/1/0.pnts
+```
+
+Unless otherwise specified, template URIs are resolved relative to the tileset.json file.
+
+![Template URI](figures/template-uri.jpg)
+
+## Content
+
+**Content** is an optional 3D model associated with a single tile, as described in the [Introduction section](https://github.com/CesiumGS/3d-tiles/tree/master/specification#introduction) of the Cesium 3D Tiles 1.0 specification, with the addition of an explicit `mimeType` to reliably identify the type of content.
```

Comment:
It's unfortunate that b3dm, i3dm, pnts, and cmpt don't have unique mime types yet which makes it hard to actually identify the type of content. CC https://github.com/CesiumGS/3d-tiles/issues/341

Reply:
On the other hand, if we convert our formats to glTF, they'd all have the same `mimeType` anyway... See https://github.com/CesiumGS/3d-tiles-next/issues/91#issuecomment-761122165. still good to have the mimeType though, but might be easier to enforce in 2.0.

Reply:
Resolving as there's nothing actionable until 2.0